CONFIG.md

```diff
@@ -57,6 +57,8 @@ ios:
   xcassetsPath: "./Resources/Assets.xcassets"
   # Is Assets.xcassets located in the main bundle?
   xcassetsInMainBundle: true
+  # Is Assets.xcassets located in a swift package?
```

Comment:
Add ... Default value is false

Reply:
# [optional] Is Assets.xcassets located in a swift package? Default value is false.